package.json

```diff
@@ -43,5 +44,14 @@
     "gulp-rename": "^1.2.2",
     "mocha": "^3.0.1",
     "run-sequence": "^1.2.2"
+  },
+  "dependencies": {
```

Comment:
these should be `devDependencies`

Reply:
Do we need to get rid of "chai" and "mocha" deps? Do the "karma-*" deps cover those?

Reply:
I just tested it, we can get rid of mocha. But chai and sinon are not covered by karma-*